src/Core/System/Snippet/SnippetValidator.php

```diff
@@ -29,7 +30,7 @@
      * @internal
      */
     public function __construct(
-        private SnippetFileCollection $deprecatedSnippetFiles,
+        private SnippetFileCollection $loadedSnippetFiles,
```

Comment:
"loaded" sounds ambiguous. It's only about our file snippets, provided by shopware, right? Maybe "bundledSnippetFiles" or so?

See this as optional. Dunno a perfect name, but maybe worth a thought 🤔

Reply:
Ah, okay